src/main/java/uk/gov/companieshouse/itemhandler/kafka/OrdersKafkaConsumer.java

```diff
@@ -238,8 +238,12 @@ protected Message createRetryMessage(String orderUri, String topic) {
         try {
             message.setValue(serializer.toBinary(orderReceived));
         } catch (SerializationException e) {
-            LOGGER.error(String.format("Error serializing message: \"%1$s\" for topic: \"%2$s\". \"%n%3$s\"",
-                    orderUri, topic, Arrays.toString(e.getStackTrace())));
+            Map<String, Object> logMap = LoggingUtils.createLogMap();
+            LoggingUtils.logIfNotNull(logMap, LoggingUtils.MESSAGE, orderUri);
+            LoggingUtils.logIfNotNull(logMap, LoggingUtils.TOPIC, topic);
+            logMap.put(LoggingUtils.EXCEPTION, e);
```

Comment:
Can we log the message offset if possible

Reply:
Good shout!